assets/js/script.js

```diff
@@ -5,8 +5,8 @@ $(document).ready(() => {
   //  var keymap = [];
   var layer = 0;
   var job_id = '';
-  var hex_stream = '';
-  var hex_filename = '';
+  var fw_stream = '';
```

Comment:
OK, if we're going to rename this, lets switch over to camel case instead of snake. s/fw_stream/fwStream/